assets/js/modules/analytics/components/setup/SetupForm.test.js

```diff
@@ -315,10 +345,6 @@ describe( 'SetupForm', () => {
 			1,
 			updateAnalytics4SettingsRegexp
 		);
-		expect( fetchMock ).toHaveFetchedTimes(
-			1,
-			getWebDataStreamsBatchRegexp
-		);
```

Comment:
Since `webDataStreamsBatch` is already populated in state, this fetch will not happen